### ISSUE TITLE
API does not allow sufficient customization

### ISSUE THREAD

**Issue by MichaelXavier** [athanclark/digestive-functors-lucid#4]
I was wondering if you'd be amenable to a breaking PR that would allow you to specify additional/overriding attributes to each exposed function in this package. With the current incarnation, you can't, for example, add a `class_` attribute to an input or specify that a form's method should be GET rather than POST. I'd be willing to do the work on this but it would pretty much break the entire API so I wanted to see how you felt about it. I'm also not 100% positive how lucid resolves attribute priority but it should always prefer the user supplied attributes on conflict.

**Comment by athanclark** [comment]
Definitely, I rarely use this package :) I would be happy to delegate ownership to you if you'd like.

**Comment by MichaelXavier** [comment]
Sounds good. It looks like I'm going to be using this in a production app. Whatever you want to do works. I'm happy to submit PRs if you want to review them. If you don't want to review them, I'd take commit access to the repo and if you don't want to deal with releasing, I'd take hackage access. With the scope of this project, I doubt it will undergo too many major upheavals beyond what I'm suggesting here, so it probably won't be too burdensome.
